components/atoms/Tooltip/tooltip.tsx

```diff
@@ -14,7 +14,7 @@ const Tooltip = ({ children, content, className, direction }: TooltipProps): JSX
   const { Portal, Root, Content, Trigger, Arrow } = TooltipPrimitive;
 
   return (
-    <Root>
+    <Root delayDuration={300}>
```

Comment:
The Radix tooltip `<Root />` component's `delayDuration` prop's default value is 700ms. See https://www.radix-ui.com/primitives/docs/components/tooltip#provider